.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+---
+
+_Please add `bug` and one or more of the following labels to your issue:_
+`frontend backend community dependencies`
```

Comment:
What does community / dependencies label mean? Also frontend/backend doesn't always have the same meaning so I don't think they are good tags (some people see browser als frontend and server as backend, others see a UI service like rocky as frontend and the rest of the services as backend). We shouldn't ask people to tag things if we don't explain what the tags mean.